config/socket/game.js

```diff
@@ -34,6 +34,7 @@ function Game(gameID, io) {
   this.questions = null;
   this.answers = null;
   this.curQuestion = null;
+  this.regionId = '1';
```

Comment:
Please assign '1', 21 and 16 to constants so that it is clear why these specific values are used.